notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>please provide link on resnet50 model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I have provided the link on resnet50 model. I have attached the paper as well. What particular link are you talking about here?

Reply:
<p>I speaking about link on pretrained model source <a href="https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html" target="_blank">https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html</a></p>